Generals/Code/GameEngine/Include/Common/TunnelTracker.h

```diff
@@ -59,7 +59,7 @@ class TunnelTracker : public MemoryPoolObject,
 	static void destroyObject( Object *obj, void *userData ); ///< Callback for Iterate Contained system
 	static void healObject( Object *obj, void *frames ); ///< Callback for Iterate Contained system
 
-#if RETAIL_COMPATIBLE_CRC
+#if RETAIL_COMPATIBLE_BUG
```

Comment:
These need to be `RETAIL_COMPATIBLE_CRC || RETAIL_COMPATIBLE_BUG`, because even if RETAIL_COMPATIBLE_BUG is turned off, it would still be CRC incompatible with Retail game.